test/RoslynAnalyzer.Tests/Verifiers/CSharpAnalyzerVerifier`1+Test.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Analyzer1.Test
+{
+    public static partial class CSharpAnalyzerVerifier<TAnalyzer>
+        where TAnalyzer : DiagnosticAnalyzer, new()
+    {
+        public class Test : CSharpAnalyzerTest<TAnalyzer, XUnitVerifier>
```

Comment:
What is this class actually trying to do (perhaps rename it) ?

Reply:
This is a standard (generated) part of the Roslyn Analyzer SDK -- we can modify it, but I just left it as is for now.